frontend/src/pages/Schedule.js

```diff
@@ -1,5 +1,51 @@
+import Page from "../components/Page";
+import content from "../assets/content/schedule.json";
+import ColorHash from "color-hash";
+
+/**
+ * @desc Displays Schedule using formatted events from JSON
+ * @author Aileen
+ */
 const Schedule = () => {
-  return <div>Schedule Page</div>;
+  const hash = new ColorHash({ lightness: 0.8 });
```

Comment:
Performance will be slightly better if you lift this out of the component; otherwise it will reinitialize on every render.